scripts/asm-installer/install_asm

```diff
@@ -867,6 +877,12 @@ install_asm(){
   sleep 1
   info "...done!"
 
+  if [[ "$DISABLE_CANONICAL_SERVICE" != 1 ]]; then
+    info "Installing ASM CanonicalService controller..."
+    retry 3 run kubectl apply -f asm/canonical-service/controller.yaml
```

Comment:
Add verification on the deployment? kubectl wait?

Reply:
Good point, PTAL.

Reply:
Also can we add a warning or error when the wait fails. Similar to what istioctl does during install.

Reply:
I agree that's nice-to-have, but we don't do that elsewhere in this script.